packages/forma-36-react-components/src/components/Workbench/Workbench.tsx

```diff
@@ -2,11 +2,13 @@ import React, { Component } from 'react';
 import cn from 'classnames';
 import Heading from '../Typography/Heading';
 import IconButton from '../IconButton';
+import Paragraph from '../Typography/Paragraph';
```

Comment:
Maybe a bit nitpicky so feel free to ignore, but I'm not sure if this makes sense as a `Paragraph` component. I'd use `Paragraphs` for longer strings of (normally multiline) text.